src/tests/ftest/io/ior_single_server.py

```diff
@@ -91,7 +91,7 @@ def test_singleserver(self):
         """
         Test IOR with Single Server config.
 
-        :avocado: tags=ior,singleserver
+        :avocado: tags=all,daosio,large,singleserver
```

Comment:
This test is actually > large as it wants 1 server and 8 clients, making it > 8 nodes which is what large is currently.